<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidgetClass</class>
 <widget class="QMainWindow" name="mainWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="m_pwdtTopWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>65</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="m_pbtnSoftWareName">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="m_pwdtButtonGroup" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background: green;</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="m_pbtnCurrentPlaying">
           <property name="text">
            <string>当前播放</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbtnLocalMusic">
           <property name="text">
            <string>本地音乐</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbtnFavoriteMusic">
           <property name="text">
            <string>我的收藏</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_pbtnSongList">
           <property name="text">
            <string>我的歌单</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="m_pwdtCurrentPlayPage">
         <widget class="QLineEdit" name="lineEdit_4">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>140</y>
            <width>331</width>
            <height>251</height>
           </rect>
          </property>
          <property name="text">
           <string>当前播放</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="m_pwdtLocalMusicPage">
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>200</y>
            <width>371</width>
            <height>231</height>
           </rect>
          </property>
          <property name="text">
           <string>本地音乐</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="m_pwdtFavoriteMusicPage">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>110</y>
            <width>261</width>
            <height>151</height>
           </rect>
          </property>
          <property name="text">
           <string>我的收藏</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="m_pwdtSongListPage">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>200</y>
            <width>301</width>
            <height>171</height>
           </rect>
          </property>
          <property name="text">
           <string>我的歌单</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="m_pwdtBottiomWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
